prow/config/jobs/istio.yaml

```diff
@@ -171,13 +171,11 @@ jobs:
       - prow/config/topology/external-istiod.json
       - test.integration.pilot.kube
     requirements: [kind]
+    resources: multicluster
```

Comment:
^^^ @stevenctl, is this needed? What is it for? I noticed it's set for all the other mc tests.

Reply:
Looks like it just increases CPU